Homework/homework3/test_task2.py

```diff
@@ -0,0 +1,7 @@
+from homework3.task2 import speed_up_calculate
+
+
+def test_speed_up_calculate():
```

Comment:
Please test calculation result, not time.  
To test time you need to have context.  So this test may fail on some machines, or even worse fail from time to time.

Reply:
Okay, understood.
But do I need somehow show that the calculations take less than a minute how it should be owing to the task?

Reply:
It's ok for me that you measure it manualy.  Time measure and unittesting are different type of testing and require different approaches.

Reply:
Keep it fast and simple.
`assert speed_up_calculate(500) ==  expected`